spec/requests/recipes_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Recipes', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/recipes/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
```

Comment:
- [ ] This test is failing as you can see below 👇. Please try to make it pass 🙏 
![image](https://github.com/Jawad-Nazari/Recipe-app/assets/46839548/060bfcaa-88a8-4512-90bf-56f7535e4573)